command/var_get.go

```diff
@@ -0,0 +1,182 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+type VarGetCommand struct {
+	Meta
+	outFmt string
+	tmpl   string
+}
+
+func (c *VarGetCommand) Help() string {
+	helpText := `
+Usage: nomad var get [options] <path>
+
+  The 'var get' command is used to get the contents of an existing variable.
+
+  If ACLs are enabled, this command requires a token with the 'var:read'
+  capability.
+
+General Options:
+
+  ` + generalOptionsUsage(usageOptsDefault) + `
+
+Read Options:
+
+  -item <item key>
+     Print only the value of the given item. Specifying this option will
+     take precedence over other formatting directives. The result will not
+     have a trailing newline making it ideal for piping to other processes.
+
+  -out ( go-template | hcl | json | none | pretty-json | table )
+     Format to render the variable in. When using "go-template", you must
+     provide the template content with the "-template" option. Defaults
+     to "table" when stdout is a terminal and to "json" when stdout is
+     redirected.
```

Comment:
This little detail turns out to be _really_ elegant for what you've done for taking input from stdin. :+1: